### DISCUSSION TITLE
Highlighting the caret line

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/textadept#400] [Q&A]
Hi,

it looks as if the method I use to highlight the caret line doesn't work anymore... or maybe it's just that I have too many things to take care of...

I used to
```
view.element_color[view.ELEMENT_CARET_LINE_BACK] = colors.bg3
view.caret_line_layer = view.LAYER_UNDER_TEXT
view.caret_line_visible = true
```
to set the caret line background...

TIA, /PA

**Comment by orbitalquark** [comment]
Try removing that last line (`view.caret_line_visible = true`). Merely setting the element style should suffice. It looks like Scintilla deprecated `caret_line_visible`, so I need to update the documentation.

**Comment by paaguti** [comment]
That’s it! Thanks a ton...
